File: Curso_logica_de_programacao/Exercicios_logica_em_C++/Crescente/Crescente.cpp
#include <iostream>

using namespace std;

int main()
{

    int x, y;
    cout << "Digite dois numeros: \n";
    cin >> x;
    cin >> y;

    while (x != y) {
        if (x > y) {
            cout << "Crescente!\n";
        } else if (x < y) {
            cout << "Decrescente!\n";
        }
        cout << "Digite outros dois numeros: \n";
        cin >> x;
        cin >> y;
    }
}

File: Curso_logica_de_programacao/Exercícios_logica_em_Python/Crescente/Crescente.py
x: int
y: int

print("Digite dois numeros: ")
x = int(input())
y = int(input())

while x != y:
    if x > y:
        print("Decrescente")
    elif x < y:
        print("Crescente")

    print("Digite dois numeros: ")
    x = int(input())
    y = int(input())